### PR TITLE
[TD-10538]<feature>taosdemo support csv ts col

### PR DESCRIPTION
**Pull request by zhaoyanggh**

### CODE REVIEW

File: src/kit/taosdemo/taosdemo.c [hunk 1]
```diff
@@ -303,6 +303,7 @@ typedef struct SSuperTable_S {
     uint64_t     lenOfTagOfOneRow;
 
     char*        sampleDataBuf;
+    bool         sampleTs;
```
Comment (sangshuduo) [inline src/kit/taosdemo/taosdemo.c]: how about useSampleTs?

File: src/kit/taosdemo/taosdemo.c [hunk 2]
```diff
@@ -5688,6 +5708,23 @@ static bool getMetaFromInsertJsonFile(cJSON* root) {
                 goto PARSE_OVER;
             }
 
+            cJSON *sampleTs = cJSON_GetObjectItem(stbInfo, "sample_ts");
```
Comment (sangshuduo) [inline src/kit/taosdemo/taosdemo.c]: use_sample_ts is better

File: src/kit/taosdemo/taosdemo.c [hunk 3]
```diff
@@ -2761,6 +2762,8 @@ static int printfInsertMeta() {
                         g_Dbs.db[i].superTbls[j].sampleFormat);
                 printf("      sampleFile:        \033[33m%s\033[0m\n",
                         g_Dbs.db[i].superTbls[j].sampleFile);
+                printf("      sampleTs:          \033[33m%s\033[0m\n",
+                        g_Dbs.db[i].superTbls[j].sampleTs ? "yes" : "no");
```
Comment (sangshuduo) [inline src/kit/taosdemo/taosdemo.c]: should prompt user disorder and disorder range will not applied if use_sample_ts be set to yes.

**Comment by zhaoyanggh** [comment]
> few comments

fixed

**Pull request merged**